rapidy/_annotation_container.py

```diff
@@ -159,13 +218,14 @@ def add_field(
             annotated_type: Type[Any],
             field_info: ParamFieldInfo,
             param_default: Any,
+            param_default_factory: Optional[NoArgAnyCallable],
```

Comment:
```suggestion
            param_default_factory: Optional[NoArgAnyCallable] = None,
```

Reply:
No need, this value is thrown higher from field_info, field_info.default_factory is by default is None. 
There is no point in throwing default_factory now. It is done for explicitness and for the future if it will be necessary to throw it otherwise.